### PR TITLE
change the Makefile to not use Black

### LINKED ISSUE #874: change the `make format` rule to not use Black
part of #871 
-change the `make format` rule to not use Black

### PR DESCRIPTION
**Pull request by avelynhc**
resolves #874
part of #865 and #852

- remove `make format` not to use black anymore
- add `make lint` to lint the project instead

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,5 +22,5 @@ deploy: build deploy-setup
 	rm .gcloudignore
 	rm main.py
 	rm requirements.txt
-format:
-	black . -l 79
+lint:
```
Comment (MaxGhenis) [inline Makefile:25]: i'd suggest we keep this as make format given your change to ensure it formats code as well as identifying lint issues (fwiw i [asked chatgpt](https://chat.openai.com/share/25354619-d9b4-4b1d-8e8b-40a69d4c1c7f) to check my intuition here)

Comment (avelynhc) [reply Makefile:27]: Got it. Just added 'black' back while keeping the `npm run lint`

**Comment by anth-volk** [comment]
Just to confirm, as part of the CI, we both format with Black and lint with Prettier?

File: Makefile (hunk 1 above)
Comment (MaxGhenis) [reply Makefile:27]: Sorry I meant to label `make format` if we're writing (not just checking). May as well keep as one command with black too:
```suggestion
    npm run lint
```

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**